lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
```

Comment:
http://stackoverflow.com/a/2051253/433558 is a solution that requires a gem (http://github.com/dazuma/versionomy), but http://stackoverflow.com/a/2051296/433558 has a pretty simple solution that gives simple comparability via `<=>` (split on `.`, `.to_i` each part, use `<=>` for comparing the arrays)